server/lib/src/constants/schema.rs

```diff
@@ -1227,6 +1242,7 @@ pub static ref SCHEMA_CLASS_DOMAIN_INFO_DL10: SchemaClass = SchemaClass {
     systemmay: vec![
         Attribute::DomainSsid,
         Attribute::DomainLdapBasedn,
+        Attribute::LdapMaxQueryableAttrs,
```

Comment:
This is the only one you need, the other versions are *past* schema versions of the domain info, and you shouldn't touch them. You only should touch the DL10 version. This will be clearer after another PR I have open will merge.